c2corg_api/views/waypoint.py

```diff
@@ -28,7 +29,11 @@
 from functools import lru_cache
 from sqlalchemy.orm import joinedload, load_only
 from sqlalchemy.orm.util import aliased
-from sqlalchemy.sql.expression import and_
+from sqlalchemy.sql.elements import literal_column
+from sqlalchemy.sql.expression import and_, union
+
+# the number of routes that are included for waypoints
+NUM_ROUTES = 30
```

Comment:
> The idea would be to do it like for the recent outings, show only a small number and give a link to see all (paginated).

That definitely makes sense for outings, because you can show only the very last ones.
But how can we figure out what routes to show in the WP page? That's not that logical I'am afraid. As for now you show the 30 most recently created routes, right?
I can check with the association if it would be OK (but I foresee some crying :P). 

> Showing 100 routes in the card layout isn't very practical.

I know :(